### PR TITLE
test(NODE-6834): fix flaky x509 test

### PR DESCRIPTION
**Pull request by durran**
### Description

Fixes the flaky x509 test.

#### What is changing?

Change tests to run against a standalone instead of sharded cluster.

##### Is there new documentation needed for these changes?

None

#### What is the motivation for this change?

NODE-6834

### Double check the following

- [x] Ran `npm run check:lint` script
- [x] Self-review completed using the [steps outlined here](https://github.com/mongodb/node-mongodb-native/blob/HEAD/CONTRIBUTING.md#reviewer-guidelines)
- [x] PR title follows the [correct format](https://www.conventionalcommits.org/en/v1.0.0/): `type(NODE-xxxx)[!]: description`
  - Example: `feat(NODE-1234)!: rewriting everything in coffeescript`
- [x] Changes are covered by tests
- [ ] New TODOs have a related JIRA ticket

### CODE REVIEW

File: .evergreen/generate_evergreen_tasks.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ BASE_TASKS.push({
   commands: [
     updateExpansions({
       VERSION: 'latest',
-      TOPOLOGY: 'sharded_cluster',
+      TOPOLOGY: 'server',
```
Comment (durran) [inline .evergreen/generate_evergreen_tasks.js:123]: There's no real reason we need to run these tests against a sharded cluster with 2 mongoses. We can only do a `w: majority` on user management commands so we were hitting an edge case where the user hadn't completely been dropped on all nodes. before the next test ran. Using a standalone now fixes this issue.

**Review by baileympearson — APPROVED** [review]
Cannot reproduce with 1000 iterations locally.  Testing with a sharded cluster locally reproduces in <10 iterations.
